addons/website_links/controller/main.py

```diff
@@ -14,7 +14,11 @@ def create_shorten_url(self, **post):
 
     @http.route('/r', type='http', auth='user', website=True)
     def shorten_url(self, **post):
-        return request.render("website_links.page_shorten_url", post)
+        return request.render("website_links.page_shorten_url", {
+            "can_generate_link_tracker": request.env['link.tracker'].has_access('create'),
```

Comment:
Just to be coherent :) 
```suggestion
            "can_create_link_tracker": request.env['link.tracker'].has_access('create'),
```